client/src/partials/login/login.js

```diff
@@ -77,12 +74,14 @@ function LoginController(AppCache, Session, Languages, Projects) {
     })
     .catch(function (response) {
 
-      // bind the $http error to the view
-      vm.error = response.data;
+      // if the user has tried too many times, display a fatal error working for
+      // ten seconds.
+      if (maxAttempts <= attempts++) {
+        return Notify.fatal('AUTH.TOO_MANY_TRYS', 10000);
```

Comment:
No issue with that logic.  I'll make the change to use `danger` uniformly
with both types or errors.